lib/reporter/process.rb

```diff
@@ -92,5 +93,17 @@ def collect_active_record_connections_stat(metric)
         end
       end
     end
+
+    def collect_failover_stats(metric)
+      dbs = RailsMultisite::ConnectionManagement.all_dbs
```

Comment:
```suggestion

```

Reply:
🤦 Thanks! This was leftover from the previous implementation